### PR TITLE
feat(django): use Python 3.11 new features

### PR DESCRIPTION
**Pull request by catcombo**

### CODE REVIEW

File: merge_checks/runner.py [hunk 1]
```diff
@@ -78,6 +78,6 @@ def run():
 
     set_commit_status(
         **(asdict(commit) | asdict(status)),
-        state=str((State.SUCCESS if checks_passed else State.FAILURE).value),
+        state=(State.SUCCESS if checks_passed else State.FAILURE),
```
Comment (zyv) [inline merge_checks/runner.py:81]: I think the parens are not needed syntactically here.

**Review by zyv — APPROVED** [review]

File: merge_checks/runner.py (hunk 1 above)
Comment (catcombo) [reply merge_checks/runner.py]: You're right! Fixed.

**Review by marns93 — APPROVED** [review]
:+1:

**Pull request merged**